nicegui/element_filter.py

```diff
@@ -118,11 +117,13 @@ def __iter__(self) -> Iterator[T]:
                 ) if content]
                 if isinstance(element, Notification):
                     element_contents.append(element.message)
-                if isinstance(element, (Select, Radio, Toggle)):
-                    options = {option['value']: option['label'] for option in element.props.get('options', [])}
-                    element_contents.append(options.get(element.value, ''))
+                if isinstance(element, ChoiceElement):
+                    is_options_dict = isinstance(element.options, dict)
+                    options = element.options.values() if is_options_dict else element.options
+                    element_contents.append(
+                        element.options.get(element.value, '') if is_options_dict else element.value)
```

Comment:
When the `ui.select` is `multiple`, `element.value` is a `list` so
```py
element.options.get(element.value, '')
```
raises
> \>                       element.options.get(element.value, '') if is_options_dict else element.value)
> E                   TypeError: unhashable type: 'list'

I added a the commit https://github.com/zauberzeug/nicegui/pull/4449/commits/996fa08b3176bcfe9913e71b2b7b7260fb0b1407 with test and fix for this use-case.